RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplateReco2D.h

```diff
@@ -0,0 +1,56 @@
+//
+//  SiPixelTemplateReco2D.cc (Version 2.20)
+//  Updated to work with the 2D template generation code
+//  2.10 - Add y-lorentz drift to estimate starting point [for FPix]
+//  2.10 - Remove >1 pixel requirement
+//  2.20 - Fix major bug, change chi2 scan to 9x5 [YxX]
+
+
+//
+//
+//  Created by Morris Swartz on 7/13/17.
+//
+//
+
+#ifndef SiPixelTemplateReco2D_h
+#define SiPixelTemplateReco2D_h 1
+
+#ifndef SI_PIXEL_TEMPLATE_STANDALONE
+#include "RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplateDefs.h"
+#include "RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplate2D.h"
+#else
+#include "SiPixelTemplateDefs.h"
+#include "SiPixelTemplate2D.h"
+#endif
+
+#define NPIXMAX 200
```

Comment:
CMS coding rules say not to use ```#define  ``` for constants

Reply:
Moved to a const unsigned int in the *.cc file.